src/horus_demod.c

```diff
@@ -180,7 +195,7 @@ int main(int argc, char *argv[]) {
 
     /* end command line processing */
 
-    hstates = horus_open(mode, Rs);
+    hstates = horus_open_advanced(mode, Rs, tone_spacing);
     horus_set_verbose(hstates, verbose);
```

Comment:
So do we need a ctest that exercises horus_open()?

Reply:
Maybe. Horus_demod doesn't use it. 

I'd say it's fairly low risk however.

Reply:
Also freedv-gui isn't using it either, as I'd still like to enable some user-customization of the options (baud rate, mask/peak estimator)

Reply:
If there is no use case for it should it be removed?